src/jimgw/samplers/blackjax/ns_aw.py

```
@@ -0,0 +1,217 @@
+    _final_state: Any
+    _nested_samples: NestedSamples
+    _n_iterations: int
+
+    def __init__(
+        self,
+        *,
+        n_dims: int,
+        log_prior_fn: Callable,
+        log_likelihood_fn: Callable,
+        log_posterior_fn: Callable,
+        config: BlackJAXNSAWConfig = BlackJAXNSAWConfig(),
```

Ruff (0.15.12)

[warning] Do not perform function call `BlackJAXNSAWConfig` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)